arch/arm/core/aarch32/cortex_m/mpu/arm_mpu.c

```diff
@@ -71,7 +71,8 @@ static inline uint8_t get_num_regions(void)
 	defined(CONFIG_CPU_CORTEX_M7)
 #include "arm_mpu_v7_internal.h"
 #elif defined(CONFIG_CPU_CORTEX_M23) || \
-	defined(CONFIG_CPU_CORTEX_M33)
+	defined(CONFIG_CPU_CORTEX_M33) || \
```

Comment:
That is also fine, but I guess it starts getting awkward; perhaps we should have hidden symbols, like ARM_MPU_V8M, and ARM_MPU_V7M, so we do not need to list all CORTEX_M variants. For future improvement.